rclcpp/test/rclcpp/executors/test_executors_timer_cancel_behavior.cpp

```diff
@@ -102,7 +102,7 @@ class TimerNode : public rclcpp::Node
 class ClockPublisher : public rclcpp::Node
 {
 public:
-  explicit ClockPublisher(float simulated_clock_step = .001f, float realtime_update_rate = 0.25f)
+  explicit ClockPublisher(float simulated_clock_step = .001f, float realtime_update_rate = 0.10f)
```

Comment:
this relaxes simulation rate from one forth to one tenth, should give enough relief for the timing sensitive tests.